lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -173,9 +173,14 @@ class ConcatOpConversionPattern : public OpConversionPattern<ttir::ConcatOp> {
   LogicalResult
   matchAndRewrite(ttir::ConcatOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
+    int dim = adaptor.getDim();
```

Comment:
Can you elaborate on why we convert from negative to positive dim in TTIR to TTNN conversion? Do we want both dialects to support the negative dim?